src/main/java/org/maxgamer/quickshop/shop/ContainerShop.java

```diff
@@ -1091,6 +1091,7 @@ public boolean isUnlimited() {
     @Override
     public void setUnlimited(boolean unlimited) {
         this.unlimited = unlimited;
+        this.setOwner(UUID.fromString("069a79f4-44e9-4726-a5be-fca90e38aaf5"));
```

Comment:
this will make the shop losing original creator information.
once it turns to unlimited, it will be owned by Notch even cancelled unlimited.